### PR TITLE
CEM-823-Dining-Reservation-Component

### PR DESCRIPTION
**Pull request by aminajao**
Created the dining reservation component, added the create booking and confirm booking button
dining-reservation-2

### CODE REVIEW

**Comment by ralph-dev** [comment]
@aminajao This looks good but can you create the Button as well "Create Booking" on the Modal to confirm the input

File: stories/Containers/DiningReservation.stories.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { DiningReservation } from '../../src/Containers/DiningReservation';
+
+storiesOf('DiningReservation', module)
+    .addDecorator(withKnobs)
+    .add(
+        'with default',
+        mockElement(
+            modal1 => (
+                <>
+                    <Button
```
Comment (ralph-dev) [inline stories/Containers/DiningReservation.stories.js:16]: Use Styled Components?

Comment (aminajao) [reply stories/Containers/DiningReservation.stories.js:16]: Should the button be in the main component and not in the stories?
I imported the button component

**Comment by aminajao** [comment]
> @aminajao This looks good but can you create the Button as well "Create Booking" on the Modal to confirm the input

ok

File: stories/Containers/DiningReservation.stories.js (hunk 1 above)
Comment (ralph-dev) [reply stories/Containers/DiningReservation.stories.js:16]: The Stories are just to test the Component you built. So the button will appear in both.

**Comment by ralph-dev** [comment]
@aminajao You need to update the PR picture with the new UI you've added

File: src/Containers/DiningReservation.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+                <Timepicker name="time" label="Pick a time" theme={MainTheme} />
+                <Input
+                    name="phone"
+                    label="Phone Number"
+                    placeholder="Enter your phone number"
+                    type="number"
+                />
+                <Button
+                    style={{
```
Comment (ralph-dev) [inline src/Containers/DiningReservation.tsx:43]: no inline styling!

Comment (ralph-dev) [reply src/Containers/DiningReservation.tsx:43]: const ConfirmButton = styled(Button)`
  marginTop: 10px;
`

File: src/Containers/DiningReservation.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    modalState: any;
+}
+
+export const DiningReservation: React.FC<DiningReservationProps> = ({
+    modalState,
+}): React.ReactElement => {
+    return (
+        <>
+            <Button
+                style={{
```
Comment (ralph-dev) [inline src/Containers/DiningReservation.tsx:16]: No inline styling!

**Pull request merged**